app/validation/validator.py

```diff
@@ -229,6 +232,26 @@ def _validate_questions(self, block_or_variant, numeric_answer_ranges):
 
         return errors
 
+    def _validate_answer_actions(self, answer):
+        errors = []
+        answer_options = answer.get('options', {})
+        for option in answer_options:
+
+            action_params = option.get('action', {}).get('params')
```

Comment:
```Python
try:
    action_params = option.get('action', {})['params']
except KeyError:
    continue
```

Reply:
This will keep the code one level less nested

Reply:
I rather keep it consistent as `if`s are already being used to check key existence.
If you really want me to reduce the indentation, I am happy to return early.

``` py
            action_params = option.get('action', {}).get('params')
            if not action_params:
                continue

            list_name = action_params.get('list_name')
```

Reply:
Yep, good compromise